docker-environment-driver/src/main/resources/isolatedDockerConfiguration/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM fedora
+MAINTAINER Marek Novotny <mnovotny@redhat.com>
+
+# install java and zip
+RUN yum -y update && yum clean all
+RUN yum -y install tar gzip java java-devel && yum clean all
+
+# set JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-openjdk
+
+## jenkins instalation
+ENV JENKINS_HOME /var/jenkins_home
+
+# Jenkins is ran with user `jenkins`, uid = 1000
+# If you bind mount a volume from host/vloume from a data container,
+# ensure you use same uid
+RUN useradd -d "$JENKINS_HOME" -u 1000 -m -s /bin/bash jenkins
+# Jenkins home directoy is a volume, so configuration and build history
+# can be persisted and survive image upgrades
+VOLUME /var/jenkins_home
+# `/usr/share/jenkins/ref/` contains all reference configuration we want
+# to set on a fresh new installation. Use it to bundle additional plugins
+# or config file with your custom jenkins Docker image.
+RUN mkdir -p /usr/share/jenkins/ref/init.groovy.d
+COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-angent-port.groovy
+ENV JENKINS_VERSION 1.554.1
+# could use ADD but this one does not check Last-Modified header
+# see https://github.com/docker/docker/issues/8331
+RUN curl -L http://mirrors.jenkins-ci.org/war-stable/$JENKINS_VERSION/jenkins.war -o /usr/share/jenkins/jenkins.war
+ENV JENKINS_UC https://updates.jenkins-ci.org
+RUN chown -R jenkins "$JENKINS_HOME" /usr/share/jenkins/ref
+# for main web interface:
+EXPOSE 8080
+# will be used by attached slave agents:
+EXPOSE 50000
+USER jenkins
+COPY jenkins.sh /usr/local/bin/jenkins.sh
+ENTRYPOINT ["/usr/local/bin/jenkins.sh"]
+# from a derived Dockerfile, can use `RUN plugin.sh active.txt` to setup /usr/share/jenkins/ref/plugins from a support bundle
+
+## end of jenkins installation
+
+USER root
+#Install SSHD
+# why? see http://blog.docker.com/2014/06/why-you-dont-need-to-run-sshd-in-docker/
+RUN yum -y install openssh-server  && yum clean all
+RUN mkdir /var/run/sshd
+
+#!!!! CHANGE ROOT PASSWORD BEFORE REAL USE !!!!
+RUN echo 'root:changeme' | chpasswd
+RUN sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+ENV NOTVISIBLE "in users profile"
+RUN echo "export VISIBLE=now" >> /etc/profile
+
+#Install maven
+# set installed Maven version you can easily change it later
+ENV MAVEN_VERSION 3.2.5
+RUN curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
```

Comment:
the same answer, rpm based installation for maven or jenkins is different from original distributed binaries.